### ISSUE TITLE
Installing Debian fails; RPC cancelled; failure in lxssmanager combase.dll

### ISSUE THREAD

**Issue by csgordon** [microsoft/WSL#5626]
# Environment

```none
Windows build number: Microsoft Windows [Version 10.0.19041.388]
Your Distribution version: Can't install to check ```lsb_release```, but latest Debian version in store (1.1.8.0)
Whether the issue is on WSL 2 and/or WSL 1: Had previously set default to 2 prior to install attempt
```
I just upgraded to version 2004 today, and was able to set the default to 2 successfully. This problem has only arisen since then.  As part of the process, I was directed to install the msi from aka.ms/wsl2kernel.

# Steps to reproduce

After installing Debian from the Microsoft Store, I try to launch it for the first time to do the distro install.  I see for a moment
```none
Installing, this may take a few minutes...
```
Then a moment later:
```none
WslRegisterDistribution failed with error 0x8007071a
Error: 0x8007071a The remote procedure call was cancelled.
```

# Logs

I can't run the distro so can't strace, but I've attached the logs requested (as a zip, since Github apparently won't let me attach
[lxcore.zip](https://github.com/microsoft/WSL/files/4957524/lxcore.zip)
 the etl files directly).  FYI, here's what shows up in the Event Viewer:
```none
Faulting application name: svchost.exe_LxssManager, version: 10.0.19041.1, time stamp: 0x7f0c4c00
Faulting module name: combase.dll, version: 10.0.19041.329, time stamp: 0x2dc62b62
Exception code: 0xc0000005
Fault offset: 0x0000000000062417
Faulting process id: 0xad4
Faulting application start time: 0x01d65fce65448751
Faulting application path: C:\WINDOWS\system32\svchost.exe
Faulting module path: C:\WINDOWS\System32\combase.dll
Report Id: ca3aa88b-a1a2-4aaa-a4fd-71e38b63d581
Faulting package full name: 
Faulting package-relative application ID: 
```
This looks similar to #4474 and #4681, but the faulting module is different here (combase.dll, rather than ntdll.dll), so I'm filing a new issue.  I see this event show up when I try to launch the Debian distro for the first time, and any time I run the ```wsl``` command (```wsl``` or ```wsl -l```).

#  Expected behavior

For Debian to install, asking me for a username and password along the way

# Actual behavior

I see for a moment
```none
Installing, this may take a few minutes...
```
Then a moment later:
```none
WslRegisterDistribution failed with error 0x8007071a
Error: 0x8007071a The remote procedure call was cancelled.
```

Thanks in advance for any help.

**Comment by therealkenc** [comment]
Thank-you for the logs; appreciated.

**Comment by csgordon** [comment]
For what it's worth, I tried updating everything again, disabling WSL, doing it over from scratch per the directions [here](https://docs.microsoft.com/en-us/windows/wsl/install-win10), and still get the exact same results today.

**Comment by csgordon** [comment]
Update: After wiping the machine and re-imaging with 20H2, and following the instructions at https://docs.microsoft.com/en-us/windows/wsl/install-win10, everything works.  I wonder if this was some interaction between the many rounds of upgrades I had done on the prior win10 installation and the wsl install process I tried last July.

**Comment by ZsomborBenedek** [comment]
For me the problem was Sophos Safeguard 8.10 as suggested here [https://github.com/microsoft/WSL/issues/4702#issuecomment-693495945](comment).
It seems to be fixed in Safeguard 8.30 (I just removed the whole thing and it works now).
